warehouse/query-core/src/test/java/datawave/query/util/ShapesIngest.java

```diff
@@ -134,6 +136,18 @@ public static void writeData(AccumuloClient client, RangeType type) throws Excep
         tops.create(SHARD_RINDEX);
         tops.create(METADATA);
 
+        IteratorUtil.IteratorScope[] scopes = IteratorUtil.IteratorScope.values();
+        for (IteratorUtil.IteratorScope scope : scopes) {
+            String name = "table.iterator." + scope.name() + ".UIDAggregator";
+            String opt = "table.iterator." + scope.name() + ".UIDAggregator.opt.*";
+
+            client.tableOperations().setProperty(SHARD_INDEX, name, "19,datawave.iterators.TotalAggregatingIterator");
+            client.tableOperations().setProperty(SHARD_INDEX, opt, "datawave.ingest.table.aggregator.KeepCountOnlyUidAggregator");
```

Comment:
this will effectively only produce shard ranges won't it?

Reply:
You're thinking of the `KeepCountOnlyNoUidAggregator`